### PR TITLE
VideoPlayer: expose stream player info to GUI 

### PR DESCRIPTION
**Pull request by popcornmix**
The first commit is to fix some incorrect/uninitialised values I found when testing.
The second adds support for omxplayer/mmal.

### CODE REVIEW

**Comment by FernetMenta** [comment]
Please rebase. I did in the meantime

File: xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp [hunk 1]
```diff
@@ -900,6 +902,9 @@ int CVideoPlayerVideo::OutputPicture(const DVDVideoPicture* src, double pts)
     return EOS_DROPPED;
   }
 
+  m_processInfo.SetVideoDimensions(pPicture->iWidth, pPicture->iHeight);
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: Those methods are not supposed to be called on every cycle. Dimensions certainly won't change in the output stage

Comment (popcornmix) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: Where is it best to call from? It is not otherwise called at all.

Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: VideoDimensions refer to video surface dimensions (uncropped). Should be set by decoder.

Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: Regarding aspect ratio I was not sure. Should this be PAR, DAR, or both?

Comment (popcornmix) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: I think DAR is more useful for displaying (e.g. DAR:1.77).
PAR can be computed fom DAR and width/height if needed.
So, either is fine (but need to make sure all codecs are consistent).

Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp]: Yes, I was thinking the same and already changed it to DAR

**Pull request merged**